Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stretch
+
+COPY . /usr/src/kafkacat
+
+RUN runDeps=''; \
```

Comment:
Is this just a matter of conventions, or does it have any practical implications?